pkgs/games/legendary-gl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, buildPythonApplication
+, requests
+}:
+
+# Requires python >= 3.8
```

Comment:
you can drop this comment once you add `disabled = ...` below

Reply:
Done :)